### PR TITLE
Update to `context.tf`. Update to Goland modules. Update GitHub Actions. Allow TF 0.13

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Update to `context.tf`
* Update to Goland modules
* Update GitHub Actions
* Allow TF 0.13

## why
* Standardization and interoperability
* Keep the module up to date

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -1,9 +1,9 @@
 terraform {
-  required_version = ">= 0.12.0, < 0.14.0"
+  required_version = ">= 0.12.0"
```
Comment (Nuru) [inline versions.tf:2]: Leave as before, with `< 0.14.0`

File: versions.tf [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
   required_providers {
-    aws   = "~> 2.0"
-    local = "~> 1.2"
-    null  = "~> 2.0"
+    aws   = ">= 3.0"
```
Comment (Nuru) [inline versions.tf:5]:
```suggestion
    aws   = ">= 2.0, < 4.0"
```

**Comment by Nuru** [comment]
@aknysh Also update all outputs to the form

```
module.this.enabled ? <value> : null
```

**Review by jamengual — DISMISSED** [review]
+1

**Pull request merged**